packages/story-editor/src/components/form/advancedDropDown/searchInput.js

```diff
@@ -143,11 +141,11 @@ const SearchInput = forwardRef(function SearchInput(
         {...rest}
       />
       <SearchIconContainer>
-        <SearchIcon />
```

Comment:
| Your branch | Staging | 
|  <img width="299" alt="Screen Shot 2021-08-16 at 4 19 19 PM" src="https://user-images.githubusercontent.com/10720454/129640862-c5ef0ab7-68fa-4e35-b986-68643497862d.png"> | <img width="313" alt="Screen Shot 2021-08-16 at 4 19 26 PM" src="https://user-images.githubusercontent.com/10720454/129640868-268aef72-e155-4f02-9561-0a5c898b945f.png"> |


littttlee itty bitty magnifying glass! 😹 


This one's pretty close - it just needs the width to be used instead of the height. 

So in the styled component `SearchIconContainer` you'll see `inputIconStyles`. `inputIconStyles` is css defined on line 36 of the same file. You'll see `> svg { ` on line 50. It has `height: 100%; width: auto;` - swap that so it's `height: auto; width: 100%;` and then go to `ClearButton` and change `> svg { height: 10px; }` to `width: 20px`. like this: 

![Screen Shot 2021-08-16 at 4 28 21 PM](https://user-images.githubusercontent.com/10720454/129641519-ec8b7210-c21a-418d-be54-1ba450feec55.png)


Maybe a little goofy but it works! 

| My branch | Figma | 
| --- | --- |
| <img width="310" alt="Screen Shot 2021-08-16 at 4 25 01 PM" src="https://user-images.githubusercontent.com/10720454/129641532-32be31e7-090a-4044-9753-da3815586d73.png"> | ![Screen Shot 2021-08-16 at 4 25 14 PM](https://user-images.githubusercontent.com/10720454/129641534-a01be134-8182-4c2b-82c5-92213951e481.png) | 

(the clear button is an out of date color, but that's out of scope.)

This is just one way to solve this one.